README.md

```diff
@@ -26,7 +26,7 @@
 
```

Comment:
- [ ] Please as the last reviewer said make sure that  external API [API Ninjas](https://api-ninjas.com/api/quotes) has been used correctly